### ISSUE TITLE
use latest version of mintty for older windows version

### ISSUE THREAD

**Issue by zevsasaki** [mintty/wsltty#350]
i am stuck at windows 1803 and the last working version of wsltty is 3.0.2
it's fine, i am using it for some time but it comes with old version of mintty and i want to use the new features

is it possible to use the latest version?

**Comment by mintty** [comment]
Yes, you can use a current cygwin build of mintty as a plug-in replacement. Just the wsltty reference in "About" would change.

**Comment by icewyvrn** [comment]
so i replaced the mintty.exe in the bin folder with the cygwin mintty,  but it does not work
![image](https://github.com/mintty/wsltty/assets/41411360/ce2f03d8-aa03-4151-8464-2fa135812216)

**Comment by mintty** [comment]
Hmm, so maybe your version was still based on the former wslbridge (not 2) version?
What's the contents of %APPDATA%/Local/wsltty/bin ?

**Comment by icewyvrn** [comment]
yea i use wsl1

![image](https://github.com/mintty/wsltty/assets/41411360/42a44d4f-f171-433a-9dff-17e20a54ea90)

**Comment by mintty** [comment]
Using the previous wslbridge is still present in the code, however, you'd need to disable the line
#define wslbridge2
in line 6770 of file winmain.c and build mintty for a matching plug-in replacement.

**Comment by icewyvrn** [comment]
okay i'll try, where is the cygwin build source? is it here? https://github.com/mintty/mintty

**Comment by mintty** [comment]
Yes, and you need a minimal cygwin installation plus packages gcc and make.

**Comment by icewyvrn** [comment]
It works! Thank you.

**Comment by mintty** [comment]
Released 3.7.2.
